datadog-logs-oci-orm/modules/policy/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "oci_identity_dynamic_group" "serviceconnector_group" {
+  #Required
+  compartment_id = var.tenancy_ocid
+  description    = "[DO NOT REMOVE] Dynamic group for log forwarding by service connector"
+  matching_rule  = "All {resource.type = 'serviceconnector'}"
+  name           = local.dynamic_group_name
+
+  #Optional
+  defined_tags  = {}
+  freeform_tags = var.freeform_tags
+}
+
+resource "oci_identity_policy" "logs_policy" {
+  depends_on     = [oci_identity_dynamic_group.serviceconnector_group]
+  compartment_id = var.tenancy_ocid
+  description    = "[DO NOT REMOVE] Policy to have any connector hub read from logging source and write to a target function"
+  name           = local.policy_name
+  statements = ["Allow dynamic-group ${local.dynamic_group_name} to read log-content in tenancy",
```

Comment:
do we not need this one as well here? `Allow group <DOMAIN>/<USER_GROUP_NAME> to read all-resources in tenancy`

Reply:
We only need to provide DatadogAuthGroup access to all resources, so that our crawlers can work. that is already being done, so there isn't any need to do the same thing here.

Reply:
The answer to this is probably "yes" but is the dynamic group here different from the one already specified in the policy stack? We already have `"Allow dynamic-group ${local.dynamic_group_name} to use fn-function in tenancy"` [for example here](https://github.com/DataDog/oracle-cloud-integration/blob/3bda8b6805fe88b4d90cac5431a86b222feae840/datadog-oci-orm/policy-setup/policy.tf#L76)

Reply:
Yes, that's right. The name of dynamic group here will be different.